tests/ethernet/test_counter_mac_addr.py

```diff
@@ -0,0 +1,58 @@
+from abstract_open_traffic_generator import flow
+from abstract_open_traffic_generator.flow import Counter
```

Comment:
let's be consistent here by simply using `flow.Counter` like all other flow components.